tutorials/distributed-calculator/react-calculator/package.json

```diff
@@ -6,14 +6,14 @@
         "server": "nodemon server.js",
         "dev": "concurrently --kill-others-on-fail \"yarn server\" \"yarn client\"",
         "start": "node server.js",
-        "buildclient": "cd client && npm install && npm install --only=dev --no-shrinkwrap && npm run build"
+        "buildclient": "cd client && npm install && npm run build"
```

Comment:
`--only=dev` is not a supported option and `--no-shrinkwrap` is obsolete

Compare to the buildclient script in tutorials/pub-sub/react-form/package.json